modules/electrophysiology_browser/jsx/components/electrophysiology_session_panels.js

```diff
@@ -0,0 +1,386 @@
+/**
+ * This file contains React component for Electrophysiology module.
+ *
+ * @author Alizée Wickenheiser.
+ * @version 0.0.1
+ *
+ */
+import React, {Component} from 'react';
+import PropTypes from 'prop-types';
+import Panel from 'jsx/Panel';
+
+
+class FilePanel extends Component {
+  constructor(props) {
+    super(props);
+    this.state = {
+      data: this.props.data,
+    };
+  }
+
+  /**
+   * Component did mount.
+   */
+  componentDidMount() {
+    console.log('componentDidUpdate check:');
+    console.log(JSON.stringify(this.state.data.downloads));
+    let downloads = [
+      'download_eeg_file',
+      'download_electrode_info',
+      'download_channels_info',
+      'download_events',
+      'download_all_files',
+      'download_fdt_file',
+    ];
+    for (let i=0; i<this.state.data.downloads.length; i++) {
+      console.log(this.state.data.downloads[i]);
+      if (this.state.data.downloads[i].file === '') {
+        document.getElementById(downloads[i]).removeAttribute('href');
+        let downloadButton = document.getElementById('btn_' + downloads[i]);
+        downloadButton.disabled = true;
+        downloadButton.style.border = '1px solid #b3b3b3';
+        downloadButton.style.color = '#b3b3b3';
+        downloadButton.style.cursor = 'not-allowed';
+        downloadButton.innerHTML = 'Not Available';
+        if (this.state.data.downloads[i].type === 'physiological_fdt_file') {
+          downloadButton.style.display = 'none';
+          let fdtTitle = document.getElementById('fdt_title');
+          fdtTitle.style.display = 'none';
+        }
+      }
+    }
+  }
+
+  /**
+   * Post-Render when we can access the DOM.
+   *
+   * @param {object} prevProps
+   * @param {object} prevState
+   *
+   */
+  componentDidUpdate(prevProps, prevState) {
+  }
```

Comment:
Why is this empty function here?

Reply:
oooops. Will be removed in the next commit